docs/release-notes.md

```diff
@@ -1,53 +1,39 @@
 # Release Notes
 
-## Key Feature Changes for 1.0
-This section outlines key features changes between versions .9 and 1.0 for [Open Active Management Technology (Open AMT) Cloud Toolkit.](Glossary.md#o) 
-
-### Additions
+## Key Feature Changes for 1.1
+This section outlines key features changes between versions 1.0 and 1.1 for [Open Active Management Technology (Open AMT) Cloud Toolkit.](Glossary.md#o) 
 
 #### [RPS](Glossary.md#r)
-- **Configuration of CIRA on Intel® AMT managed devices:** Enables managed devices to connect to the MPS.
-- **Un-configuration Intel® AMT devices:** Removes all Intel® AMT configuration settings and resets Intel AMT to an [*nprovisioned](Glossary.md#p) state.
-- **REST APIs:** Includes Create, Read, Update, Delete AMT Profiles, Domain Profiles, and CIRA Profiles.
+- **Network Configuration on Intel&reg; AMT devices:** The AMT Profile has a setting to select configuring Intel&reg; AMT devices to use DHCP or Static IP.  When set for static IP, Intel&reg; AMT will sync with the Host OS static IP address.  Static IP Address is not supported with CIRA profiles.
+- **MEBx Password configuration:** The AMT Profile now supports configuration options for setting either a defined or randomized MEBx password on Intel&reg; AMT devices.
+- **CORS support:** Added support for Cross-Origin Resource Sharing (CORS).
 
 #### [RPC](Glossary.md#r)
-- **microLMS in the RPC executable:** Eliminates the need to run a separate LMS when trying to configure Intel® AMT.
-- **CentOS*:** Enables CentOS on RPC. Linux build environment is still Ubuntu* 18.04.
-- **Intel® AMT information commands:**  Provide feedback to the local command prompt showing status information of the Intel® AMT device.
+- **Improved Link Status when using --amtinfo:** Fixed link status not being set.
+- **Add/update DNS Suffix (OS), Hostname (OS), FQDN (AMT), and DNS Suffix (AMT):** Improved pulling metadata from Intel&reg; AMT and Host OS.
+- **Improved build scripts for Windows and Linux:**  Build scripts have been updated so that the build time and executable size is more consistent between Windows and Linux.
+- **CentOS 7 Support:** Build steps added to support CentOS 7.
```

Comment:
Should add a bullet for CentOS 8 support as well, since the documentation calls it out specifically.

Reply:
We had CentOS support in 1.0 release notes.  No version called out specifically, but yes, it was CentOS 8.  1.1 adds support for CentOS 7.